pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -1851,6 +1865,14 @@ void run() throws PulsarAdminException {
                     maxBlockSizeInBytes, readBufferSizeInBytes, offloadAfterThresholdInBytes,
                     offloadAfterElapsedInMillis, offloadedReadPriority);
 
+            if (offloadMethod != null) {
```

Comment:
Don't we need to set a default value to the `ledger-base` for the offload method?

Reply:
It has a default value at the `OffloadPolicies` definition